src/ElementsRenderer.js

```diff
@@ -5,18 +5,8 @@ const propTypes = {
 };
 
 function accumulateElement(children, element) {
-  if (element === undefined) {
-    // No element specified for this route.
-    return children;
-  }
-
-  if (element === null) {
```

Comment:
Can't hit this any more... if any elements are `undefined` (i.e. during loading), `resolveElements` will yield `undefined`... unless the user does something dumb